crates/cli-support/src/js/mod.rs

```diff
@@ -388,7 +388,7 @@ impl<'a> Context<'a> {
                 js.push_str("let wasm = undefined;\n");
                 init = self.gen_init(needs_manual_start, None)?;
                 footer.push_str(&format!(
-                    "{} = Object.assign(init, {{ initSync }}, __exports);\n",
+                    "{} = Object.assign(wasm_bindgen_init_input, {{ initSync }}, __exports);\n",
```

Comment:
I'm not sure what `init_input` is supposed to mean. Could it just be `wasm_bindgen_init` instead?

Also, all the prefixed names like this that we already have in `wasm-bindgen` are prefixed with `__wbg_`, not `wasm_bindgen_`, so it might be better to call it `__wbg_init` for consistency.

Reply:
agreed, done!